rust/src/ldap/ldap.rs

```diff
@@ -296,10 +294,15 @@ impl LdapState {
             match ldap_parse_msg(start) {
                 Ok((rem, msg)) => {
                     let response = LdapMessage::from(msg);
+                    println!(
+                        "lolr {} {}/{}",
```

Comment:
lolr

Reply:
Argh, is there a good way to catch these on CI ? Or should I just add a personal `git grep lol` in my pre-push git hook ?

Reply:
I guess any stdout/stderr w/o the regular suri prefix would be one way.

A println grep could be another, if we have no real users.

Grepping for common patterns could be another.

None of them seem completely reliable or robust I guess.

Reply:
> A println grep could be another, if we have no real users.

Tried that and we have 3 occurrences, 2 in tests...